tests/testthat/test-load.r

```diff
@@ -0,0 +1,10 @@
+context("Loading")
+
+test_that("Package root is working directory when loading", {
+  expect_message(load_all("testLoadDir"), "[|].*/testLoadDir[|]")
+})
+
+test_that("Loading a package subdirectory throws an error", {
```

Comment:
I think the goal was to make `load_all()` just work for people, even before they'd formally made a package.